src/gui/Src/BasicView/HexDump.cpp

```diff
@@ -642,9 +644,9 @@ void HexDump::keyPressEvent(QKeyEvent* event)
                 printDumpAt(offsetVa, false);
         }
     }
-    else if(modifiers == Qt::ShiftModifier)
+    else
     {
-        //TODO
+        AbstractTableView::keyPressEvent(event);
     }
     /*
         Let's keep the old code for a while until nobody remembers previous behaviour.

```

Comment:
Feel free to get rid of this too

Reply:
Not sure if this is useful, now it's removed.

Reply:
> Feel free to get rid of this too

> Not sure if this is useful, now it's removed.

Doesn't work after removing this, my fault, sorry.